ivy/functional/frontends/tensorflow/ragged/ragged.py

```diff
@@ -0,0 +1,135 @@
+import ivy
+
+
+# TODO: Align behavior with tensorflow, modify so that the elements of the raggedTensor
+#  object are of type EagerTensor
+# ensure that the values and row_splits are of type EagerTensor too
+# add more initializer methods
+
+
+class RaggedTensor:
+    def __init__(self, values, row_partition, internal=False, data=None):
+        if not internal:
+            raise ivy.exceptions.IvyException(
+                "RaggedTensor constructor is private; please use one of the "
+                "factory methods instead "
+                "(e.g., RaggedTensor.from_row_lengths())"
+            )
+        self._values = values
+        self.data = data
```

Comment:
If `self.data` stores an ivy array/container, could we name it as such. Like how we renamed the data attribute in the frontend array/tensor classes to ivy_array

Reply:
yes we can change this when we have NestedArray class implemented.